script/Arcueid/Day 14/14_00_ARC14_1.txt

```diff
@@ -1235,6 +1235,7 @@ When I come to my senses, I see that the sun has long since set.
 [sha:3b65084d398f71405d7acaf8c9086a64945887ef]{
 -- Page 74, Offset 36986.
 -- 真紅だった空気は消えて、空は青く青く染まっていた。
+// group pr check: cg is not blue at all										
```

Comment:
no, it's not that it isn't blue, it's that it's not a "deep" blue. I'd say it's somewhere around #084d78.

Reply:
> dyed a clear and brilliant shade of blue

Don't have my save on this pc yet but I don't feel like deep blue is _that_ off